schema/authorisation.js

```diff
@@ -1,4 +1,5 @@
 const BaseModel = require('./base-model');
+const aslConstants = require('@asl/constants');
```

Comment:
I'd probably not bother with the `aslConstants` variable since we are generally only ever using one of them, and the context is unimportant.

Instead do:

```js
const { authorisationTypes } = require('@asl/constants');
```